app/controllers/accounts/connected_accounts_controller.rb

```diff
@@ -13,6 +13,7 @@ def show
         sp_name: decorated_session.sp_name,
         decorated_user: current_user.decorate,
         locked_for_session: pii_locked_for_session?(current_user),
+        account_reset_requested: current_user&.account_reset_request,
```

Comment:
similar to https://github.com/18F/identity-idp/pull/6513#discussion_r910307525, I think we want just the boolean here:

```suggestion
        account_reset_requested: current_user&.account_reset_request.present?,
```